.changeset/seven-ducks-attend.md

```diff
@@ -0,0 +1,6 @@
+---
+"@effection/core": minor
+"effection": minor
+---
+
+Add EffectionError and wrap errors in this error automatically
```

Comment:
Collect trace of effection operations and propagate them along with the raised error